actors/common/src/main/java/org/sunbird/metrics/actors/CourseMetricsActor.java

```diff
@@ -170,7 +176,9 @@ private String getCourseNameFromBatch(Map<String, Object> courseBatchResult) {
 
   @SuppressWarnings("unchecked")
   private void courseProgressMetrics(Request actorMessage) {
-    ProjectLogger.log("CourseMetricsActor-courseProgressMetrics called");
+    ProjectLogger.log(
+        "CourseMetricsActor:courseProgressMetrics CourseMetricsActor-courseProgressMetrics called",
```

Comment:
From:
"CourseMetricsActor:courseProgressMetrics CourseMetricsActor-courseProgressMetrics called"

To:
"CourseMetricsActor: courseProgressMetrics called."